DistributionUpdate/PackageUpdate/PackageInfoTools.g

```diff
@@ -336,23 +336,34 @@ UpdatePackageInfoFiles := function(pkgdir)
                 "\n  This is not allowed, so the info file will not be changed\n");
           has_error := true;
         fi;
-        if IsBound(infon.Date) then
-          date := infon.Date;
-          if not IsString(date) and Length(date) = 10 and date{[3,6]} = "//" and
```

Comment:
Huh -- am I confused, or is this old code simply wrong? Seems as if parenthesis are missing after `not` and before `then` ... ?

Reply:
Yes I'm pretty sure that is wrong.